docs/index.md

```
@@ -31,43 +31,43 @@
+    - Create `.devcontainer/.env` with your tokens:
     ```env
-    GIT_USER=VotreNom
-    GIT_EMAIL=votre@email.com
+    GIT_USER=YourName
+    GIT_EMAIL=your@email.com
     GITHUB_TOKEN=ghp_xxx
     ```
 
-4. **Commencer à travailler**
+4. **Start working**
     ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)